### DISCUSSION TITLE
Logbook MS with FBW aircraft

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4547] [General]
Hi developers,

interresting thing I saw today. In the loogbok of MS 2020 is always a United A 320. I flew different liveries. Coulb it be a bug of MS or FBW ?

![grafik](https://user-images.githubusercontent.com/76531893/115864386-0e990d00-a437-11eb-943e-06348f303b00.png)

greets Klaus

**Comment by ExampleWasTaken** [comment]
Everything about the logbook is Asobo.
